package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@newrelic/security-agent",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
You shouldn't be updating the version manually. This is being done in the npm-publish.yml workflow